### ISSUE TITLE
Converting API requests to the new Twitch API

### ISSUE THREAD

**Issue by mvarendorff** [cavariux/twitchirc#28]
There is still plenty of time left but better be safe than sorry.
Twitch API v3 and v5 are already deprecated and the endpoints will be turned off at the end of 2018 with only the new Twitch API remaining.
Meaning that the requests currently implemented will not work anymore.
Hence they have to be changed to use the new API.

**Comment by lmariscal** [comment]
I haven't touched java in years haha, but will do my best to help you with this task.

**Comment by mvarendorff** [comment]
Yeah I will see when I have time/motivation to do it. Currently I am working fulltime again and the semester starts again in March so I guess that is when I might do some more stuff here again ^^
